providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/grpc/GrpcConnectorTest.java

```diff
@@ -123,80 +117,6 @@ void initialization_fail_with_timeout() throws Exception {
         assertThrows(RuntimeException.class, connector::initialize);
     }
 
-    @Test
```

Comment:
@Kavindu-Dodan maybe this is a dumb question - could we keep this tests, but run them conditionally if epoll IS available?

Reply:
We could perhaps use: https://www.baeldung.com/junit-5-conditional-test-execution#2-conditions

Reply:
@toddbaert  added them with conditional execution `@EnabledOnOs(OS.LINUX)` - https://github.com/open-feature/java-sdk-contrib/pull/404/commits/35d293bbea9d60b13bd82c9e8483388629fc4d46

This is ideal solution and thanks for recommending this :)